lib/requester.js

```diff
@@ -22,57 +22,63 @@ module.exports = function(requestObject, debug, cb){
   method = requestObject.method && requestObject.method.toLowerCase(),
   headers = headersToRequestFormat(requestObject.headers),
   json, rawHttpRequest, rawHttpResponse, r;
-  
+
   // Ensure we convert out of a mongoose document
   if (requestObject.toObject){
     requestObject = requestObject.toObject();
   }
-  
+
   if (_.isFunction(debug)){
     cb = debug;
     debug = false;
   }
-  
+
   if (!requestObject.url || !method){
     return cb({ code : 400, message : 'All requests must contain a URL and Method' });
   }
-  
+
   if (!request.hasOwnProperty(method)){
     return cb({code : 405, message : 'Method ' + method + ' is not supported'});
   }
-  
-  if (requestObject.body && _.isObject(requestObject.body)){
+
+  if(_.isObject(requestObject.body))
+  {
     requestObject.body = JSON.stringify(requestObject.body);
   }
-  
+
   if (headers.hasOwnProperty('content-type') && headers['content-type'] === 'application/json'){
```

Comment:
We first stringify `requestObject.body` if  it's an object and then, if content-type equals json, we convert it back to object? Maybe that can be reduced to one step?